packages/components/src/styles/components/time.scss

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// TIME COMPONENT
+//
+
+.hds-time-range {
+  display: inline-flex; // remove excess space between time range elements
+  gap: 2px; // add a bit of space around en dash so it doesn't touch some number characters
```

Comment:
@majedelass I added just a bit of spacing around the en dashes in date ranges to prevent them from actually touching text. I don't have a way of adding the same spacing with tooltip content however as those use only plain text (no html).

Reply:
Btw, sometimes "thin spaces" or "hair spaces", which are actual typed characters, are added around en & em dashes. See: https://en.wikipedia.org/wiki/Whitespace_character#Hair_spaces_around_dashes

I actually experimented with using these instead of CSS styling for spacing which worked, but our code linter currently doesn't allow them and throws an error.

Reply:
Ok, ill make sure the content reflects this. Thank you!